### DISCUSSION TITLE
Premières analyses des DUMPS - eventAttendanceMode ou EventStatus

### DISCUSSION THREAD

**Discussion by GenFab28** [culturecreates/artsdata-planet-lavitrine#28] [Modelling discussions]
En important les DUMPS dans le système de La Vitrine, voici les observations sur certaines inconsistances / à discuter voir si on peut améliorer les importations

Certaine représentations n’ont pas eventAttendanceMode ou de eventStatus : dans ces cas on prend “Scheduled” et “offline” en valeur par default, mais est-ce possible de le populer par défaut de votre côté s'i l'information ne se trouve pas sur les sites?

        {
          "@id": "http://kg.footlight.io/resource/ccat-qc-ca_m0ts-premiers-par-ak-entrepot-laurence-henry#2024-05-20T100000-0400",
          "type": "Event",
          "name": {
            "fr": "M0ts Premiers par AK Entrepôt Laurence Henry - Conseil de la culture de l'Abitibi-Témiscamingue"
          },
          "startDateTime": "2024-05-20T10:00:00-04:00",
          "offers": {
            "@id": "http://kg.footlight.io/resource/ccat-qc-ca_m0ts-premiers-par-ak-entrepot-laurence-henry#2024-05-20T100000-0400_WrappedOffer",
            "type": "Aggreg 
            
            
@saumier   @christianroy

**Comment by saumier** [comment]
@GenFab28 @christianroy Yes. The data pipeline that exports events from Artsdata can transform the data. I can add  “Scheduled” et “offline” when they are missing. Please confirm.

**Reply by GenFab28** [reply]
@saumier "Scheduled" par défaut lorsque manquant. Merci

**Reply by saumier** [reply]
@GenFab28 Just to be sure, you'd like me to leave the eventAttendanceMode untouched? Or should I hardcode the eventAttendanceMode to "offline"?

**Reply by GenFab28** [reply]
@saumier offline for eventAttendanceMode. and Scheduled for the status
Thanks
